src/icons/two-toned/IconCollectionItemLink.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import type { TwoTonedIcon } from '../flowTypes';
+import { bdlGray, white } from '../../styles/variables';
+
+const IconCollectionItemLink = ({ className = '', height = 24, title, width = 24 }: TwoTonedIcon) => (
```

Comment:
I would start defaulting new icons to 32 32 also, @booc0mtaco ?

Reply:
most of the two-toned default to 24 right now

Reply:
but yeah defaulting to 32px seems reasonable seeing as that's the viewbox

Reply:
i think using default values equal to the viewbox could work. getting into the habit of specifying the width/height where the icons are used would make future refactoring easier, as well